packages/components/src/utils/frameProps.tsx

```diff
@@ -45,11 +45,21 @@ type TransientFrameProps = TransientProps<FrameProps>;
 const getValueWithUnit = (value: string | number) =>
     typeof value === 'string' ? value : `${value}px`;
 
+export const pickAndPrepareFrameProps = (
+    props: Record<string, any>,
+    allowedFrameProps: Array<string>,
```

Comment:
How about this for a type-safety improvmenet? :thinking: 
```suggestion
    props: Partial<FrameProps>,
    allowedFrameProps: Array<FramePropsKeys>,
```

Reply:
Let's keep it. As discussed in person, there is a valid requirement to pass any props through this function.
I'd keep the `allowedFrameProps: Array<FramePropsKeys>,` though..